### ISSUE TITLE
Notice after plugin activation | AMP_QUERY_VAR

### ISSUE THREAD

**Issue by moldcraft** [ahmedkaludi/accelerated-mobile-pages#84]
```
Notice: Use of undefined constant AMP_QUERY_VAR - assumed 'AMP_QUERY_VAR' in /.../wp-content/plugins/accelerated-mobile-pages/accelerated-moblie-pages.php on line 18

Notice: Use of undefined constant AMP_QUERY_VAR - assumed 'AMP_QUERY_VAR' in /.../wp-content/plugins/accelerated-mobile-pages/accelerated-moblie-pages.php on line 19
```

v0.8.1

**Comment by moldcraft** [comment]
On frontend I get

```
Notice: Undefined index: opt-media in /.../wp-content/plugins/accelerated-mobile-pages/templates/header.php on line 7
```

**Comment by ahmedkaludi** [comment]
We will look into this and fix it in the upcoming updates.

For now, you can set the debug 'false' in wp-config.php to hide them from the website.

Thank you so much for using our plugin and reporting the error.

Regards,
Ahmed

**Comment by technosailor** [comment]
This really should get fixed soon, in my opinion.

**Comment by ahmedkaludi** [comment]
Hey Aaron @technosailor 

Yes! We're actually working on fixing this. we will fix this in the next update as soon as possible.

Thank you so much for helping us improve the plugin with your feedback. I have a great respect for the work you guys are doing at @10up ! :)


Regards,
Ahmed Kaludi

**Comment by MohammedKaludi** [comment]
All the above-mentioned warnings have been fixed in #300 and will be pushed in the update V0.9.33

Thank you @moldcraft and @technosailor for your feedback.
